README.md

```diff
@@ -25,6 +25,7 @@ Use new SQLite features:
 - **[JSON1 extension](https://www.sqlite.org/json1.html)**
 - **[Common Table expressions](https://www.sqlite.org/lang_with.html)**
 - **[Indexes on expressions](https://www.sqlite.org/expridx.html)**
+- **[Full Text Search 5](https://www.sqlite.org/fts5.html)**
```

Comment:
I don't believe this is accurate unless some further action is taken in the build system here?

https://www.sqlite.org/fts5.html

> 2.1. Building FTS5 as part of SQLite
As of version 3.9.0 (2015-10-14), FTS5 is included as part of the SQLite amalgamation. It is disabled by default. If using the two autoconf build system, it is enabled by specifying the "--enable-fts5" option when running the configure script.

Or, if sqlite3.c is compiled using some other build system, by arranging for the SQLITE_ENABLE_FTS5 pre-processor symbol to be defined.